### ISSUE TITLE
library incompatible with @angular/cli 1.7.1

### ISSUE THREAD

**Issue by mwawrusch** [google/material-design-icons#737]
After upgrading to 1.7.1 I get the following when compiling with AOT:

```
ERROR in ./node_modules/raw-loader!./node_modules/postcss-loader/lib??embedded!./node_modules/material-design-icons/iconfont/material-icons.css
(Emitted value instead of an instance of Error) CssSyntaxError: /Users/mwawrusch/Documents/oneclickico/oneclickico-web/node_modules/material-design-icons/iconfont/material-icons.css:9:12: Can't resolve 'MaterialIcons-Regular.woff)%20format(%27woff%27' in '/Users/mwawrusch/Documents/oneclickico/oneclickico-web/node_modules/material-design-icons/iconfont'
```

**Comment by Dunos** [comment]
This error is what I reported in [#733](https://github.com/google/material-design-icons/issues/733)

**Comment by mwawrusch** [comment]
Sorry Dunos, missed that.

**Comment by Dunos** [comment]
This has been fixed un CLI 1.7.2 anyway
